.ai/spec/what/plugin-api.md

```
@@ -0,0 +1,101 @@
+
+3. The popover is launched using a stable modal ID to prevent duplicate
+   instances.
+
+### useOpenOLS Public API
+
+4. The `useOpenOLS` hook must be exposed as a console action provider so
+   other plugins and console pages can programmatically open the OLS chat.
+
+5. The hook returns a callback function with the following signature:
+
+   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)